wp-admin/js/widgets/media-widgets.js

```diff
@@ -141,6 +236,21 @@ wp.mediaWidgets = ( function( $ ) {
 		fetchSelectedAttachment: function fetchSelectedAttachment() {
 			var control = this, attachment;
 
+			// This is an embed (by URL) image if the url is set and the attachment_id is 0.
+			if ( 0 === control.model.get( 'attachment_id' ) && control.model.get( 'url' ) ) {
+
+				// Construct an attachment model with the data we have.
+				attachment = new wp.media.model.Attachment( control.model.attributes );
+
+				// Once the widget renders, inject the preview.
+				_.defer( function() {
```

Comment:
This `defer` seems a bit brittle and it may indicate we should improve something with how `selectedAttachment` is used.